### PR TITLE
Update minicraftcz

### PR DESCRIPTION
**Pull request by TestrGames**
## Type of change

- [ ] Add new directory for a server
- [x] Update directory of a server
- [ ] Remove directory of a server
- [ ] Documentation Update

## Checklist

- [x] I have read the [README](https://github.com/LabyMod/server-media/blob/master/README.md) doc
- [x] I have compressed the images appropriately (e.g. on https://tinypng.com)
- [x] I have created the manifest.json with the required values

### CODE REVIEW

File: minecraft_servers/minicraftcz/manifest.json [hunk 1]
```diff
@@ -12,7 +12,7 @@
     "social": {
       "web": "https://minicraft.cz",
       "web_shop": "https://store.minicraft.cz",
-      "web_support": "https://www.minicraft.cz/kontakt",
+      "web_support": "https://www.minicraft.cz/forum",
```
Comment (maaaathis) [inline minecraft_servers/minicraftcz/manifest.json:15]: `web_support` cannot be a forum; use a faq or contact page instead

File: minecraft_servers/minicraftcz/manifest.json [hunk 2]
```diff
@@ -28,6 +28,20 @@
        "background": "#4f3702",
        "text": "#FFFFFF"
     },
+    "gamemodes": {
+      "getdown": {
+        "name": "Survival",
```
Comment (maaaathis) [inline minecraft_servers/minicraftcz/manifest.json:33]: Key and name doesn't match

File: minecraft_servers/minicraftcz/manifest.json [hunk 3]
```diff
@@ -28,6 +28,20 @@
...
+        "command": "/tpsurvival"
+      },
+      "survival": {
+        "name": "OneBlock",
```
Comment (maaaathis) [inline minecraft_servers/minicraftcz/manifest.json:37]: Key and name doesn't match

File: minecraft_servers/minicraftcz/manifest.json [hunk 4]
```diff
@@ -28,6 +28,20 @@
...
+        "command": "/tponeblock"
+      },
+      "gungame": {
+        "name": "SkyBlock",
```
Comment (maaaathis) [inline minecraft_servers/minicraftcz/manifest.json:41]: Key and name doesn't match

**Review by maaaathis — CHANGES_REQUESTED** [review]

**Comment by DoJapHD** [comment]
Server is offline; Reopen PR once server is back online

**Comment by TestrGames** [comment]
hello, the server is not offline. We operate only in the Czech Republic and Slovakia. We have therefore blocked any access from the USA and similar countries to the server, it makes no sense for us to allow it.

**Pull request closed without merging**